app/helpers/application_helper/button/cloud_volume_new.rb

```diff
@@ -0,0 +1,18 @@
+class ApplicationHelper::Button::CloudVolumeNew < ApplicationHelper::Button::Basic
+  def calculate_properties
+    super
+    if disabled?
+      self[:title] = _("No cloud providers support creating cloud volumes.")
+    end
+  end
+
+  def disabled?
+    ems_clouds = EmsCloud.all
+    # if any connected provider supports this action,
+    # we can enable the button.
+    ems_clouds.each do |ems|
+      return false if CloudVolume.class_by_ems(ems).supports_create?
+    end
+    true
```

Comment:
@mzazrivec The problem is that the BZ specifically asks that the option be disabled.  This also matches what happens in other parts of the UI when X doesn't support option Y.  I think it might be a really bad thing to introduce another pattern here, as that can be highly disorienting to the user... ?